powerhome/playbook
# [PLAY-216] - TS Conversion of Date Stacked kit + Jest Test

#### Screens

![Playbook-Design-System (8)](https://user-images.githubusercontent.com/73710701/190201128-8e4a82ec-d20a-4788-b8b3-ada208e5c9e5.png)

#### Breaking Changes

No breaking changes,  converts Date Stacked Kit to Typescript and adds Jest test.

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-216)

#### How to test this

Review in milano env. Navigate to date stacked kit and verify that all examples work as expected.

#### Checklist:
